pkgs/top-level/all-packages.nix

```
@@ -803,6 +803,8 @@ in
 
   claws = callPackage ../tools/misc/claws { };
 
+  cloud-custodian = callPackage ../tools/networking/cloud-custodian { };
```

alternative
```suggestion
  cloud-custodian = python3Packages.callPackage ../tools/networking/cloud-custodian { };
```

then declare all your needed packages like you would as if it were in python-modules